domains/todoo.json

```diff
@@ -7,6 +7,6 @@
         "twitter": "ce3s"
     },
     "record": {
-        "CNAME": "iibeboq.github.io/todo"
+        "CNAME": "iibeboq.github.io"
```

Comment:
404?

Reply:
> 404?

Bc its with /todo not iibeboq.github.io

Reply:
oh

Reply:
> oh

Now i will do anything again? Make it back to /todo?

Reply:
Keep it like this. It will work even though the repo is called `todo`.